packages/server/lib/middleware/access.middleware.ts

```diff
@@ -431,6 +431,51 @@ export class AccessMiddleware {
         }
     }
 
+    /**
+     * Test authentication that accepts both secret key and session authentication
+     * This allows tests to use either authentication method
+     */
+    async testAuth(req: Request, res: Response<any, RequestLocals>, next: NextFunction) {
```

Comment:
can we just add a safety net "if (vitest)" or something like this to prevent accidental use